public/app/features/explore/Logs/Logs.tsx

```diff
@@ -174,16 +174,16 @@ class UnthemedLogs extends PureComponent<Props, State> {
     if (this.cancelFlippingTimer) {
       window.clearTimeout(this.cancelFlippingTimer);
     }
-    // Delete url state on unmount
-    if (this.props?.panelState?.logs?.columns) {
-      delete this.props.panelState.logs.columns;
-    }
-    if (this.props?.panelState?.logs?.refId) {
-      delete this.props.panelState.logs.refId;
-    }
-    if (this.props?.panelState?.logs?.labelFieldName) {
-      delete this.props.panelState.logs.labelFieldName;
-    }
+
+    dispatch(
```

Comment:
hmm, we might want to gate this to only update when columns, labels, etc are set